neurobooth_os/transfer_data.py

```diff
@@ -0,0 +1,32 @@
+"""
+    Moves data from local storage to network storage
+"""
+import subprocess
+import sys
+
+from neurobooth_os import config
+
+server_names = ("presentation", "acquisition", "control")
+
+if len(sys.argv) < 2:
+    raise Exception("Server name is a required argument for this script.")
+
+server_name = sys.argv[1]
+
+if server_name is None or server_name == '':
+    raise Exception("You must provide a server name to transfer data")
+
+if server_name not in server_names:
+    raise Exception(f"The server name argument must be one of {server_names}.")
+
+destination = config.neurobooth_config["remote_data_dir"]
```

Comment:
Add checks to ensure that the destination and source exist and are valid directories? I.e., os.path.exists and os.path.isdir. I would also use os.path.abspath to get an absolute path for the (potentially relative) paths and log the result to help with debugging.

Reply:
Could do some of this in the JSON parsing code so that everything shares these checks.

Reply:
done